## Description: Let adb detect devices from BQ.
## Origin/Author: Jani Monoses <jani.monoses@canonical.com>
Index: android-tools-4.2.2+git20130218/core/adb/usb_vendors.c
===================================================================
--- android-tools-4.2.2+git20130218.orig/core/adb/usb_vendors.c
+++ android-tools-4.2.2+git20130218/core/adb/usb_vendors.c
@@ -131,6 +131,8 @@
 #define VENDOR_ID_KOBO          0x2237
 // Teleepoch's USB Vendor ID
 #define VENDOR_ID_TELEEPOCH     0x2340
+// BQ's USB Vendor ID
+#define VENDOR_ID_BQ            0x2a47
 
 
 /** built-in vendor list */
@@ -183,6 +185,7 @@ int builtInVendorIds[] = {
     VENDOR_ID_YULONG_COOLPAD,
     VENDOR_ID_KOBO,
     VENDOR_ID_TELEEPOCH,
+    VENDOR_ID_BQ,
 };
 
 #define BUILT_IN_VENDOR_COUNT    (sizeof(builtInVendorIds)/sizeof(builtInVendorIds[0]))
